test/functional/dataElements/xdmObject/helpers/platformMocks.js

```diff
@@ -314,9 +308,6 @@ export const schemasMetaPagingTitles = [
 export const schemasMetaPagingMock = RequestMock()
   .onRequestTo({
     url: SCHEMAS_ENDPOINT_REGEX,
-    headers: {
-      "x-sandbox-name": "prod"
-    },
```

Comment:
This wasn't actually doing anything. `headers` isn't a valid option for `onRequestTo`.